release-notes/swan-lake-beta2-release-note.md

```diff
@@ -36,6 +36,27 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+The `Configurable` feature is improved to support variables with the `union` types through the TOML syntax. 
+
+Here are some examples with `union` types.
```

Comment:
Do we need this? We've just added the examples for others IINM.

Reply:
This is also the same as beta 1 release note when we support enums.

Reply:
I guess it is okay, but we haven't said anything like this for the rest. Was wondering if it is inconsistent.